tests/test_create_diagraph.py

```diff
@@ -2,38 +2,20 @@
 from os.path import abspath, dirname, isfile
 
 import pytest
+from django.core.management import call_command
 from django.core.management.base import CommandError
-from django.test.utils import override_settings
 
 from viewflow_extensions.management.commands import create_diagraph
 
 FLOW_NAME = 'TestFlow'
-APP_NAME = 'tests.testapp'
+FILE_PATH = 'tests.testapp.flows'
 
 
-def test_create_diagraph_debug_false():
-    """Test when settings.DEBUG == False"""
-    cmd = create_diagraph.Command()
-    options = {
-        'app': [APP_NAME, ],
-        'flow': [FLOW_NAME, ],
-        'svg': None
-    }
-    cmd.handle(**options)
-    dot_file_path = os.path.join(dirname(dirname(abspath(__file__))),
-                                 '{}.{}'.format(APP_NAME, FLOW_NAME))
-    svg_file_path = os.path.join(dirname(dirname(abspath(__file__))),
-                                 '{}.{}.svg'.format(APP_NAME, FLOW_NAME))
-    assert isfile(dot_file_path) is False
-    assert isfile(svg_file_path) is False
-
-
-@override_settings(DEBUG=True)
-def test_create_diagraph_debug_true():
+def test_create_diagraph():
     """Test when settings.DEBUG == False"""
     cmd = create_diagraph.Command()
     options = {
-        'app': [APP_NAME, ],
+        'path': [FILE_PATH, ],
```

Comment:
i left it here because call_command doesn't return anything and this way I can test the output